PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-database/src/main/java/com/microsoft/azure/toolkit/intellij/database/component/UsernameComboBox.java

```diff
@@ -32,4 +36,10 @@ public void setServer(IDatabaseServer<?> server) {
     public boolean isRequired() {
         return true;
     }
+
+    @Nonnull
+    @Override
+    protected List<ExtendableTextComponent.Extension> getExtensions() {
```

Comment:
shall we keep refresh for username combobox? Which may be affected by users' operation outside toolkit

Reply:
fixed.